.github/workflows/dev-infra.yml

```diff
@@ -0,0 +1,15 @@
+name: DevInfra
```

Comment:
should we name this slash commands instead? I imagine if we add more actions part of dev-infra, all could fall into this workflow file, but that is not always possible/desired (if different events are used)